SQLQuery_9.sql

```diff
@@ -0,0 +1,21 @@
+CREATE PROCEDURE spSummariseEpisodes
+AS
+BEGIN
+    -- The 3 most frequently-appearing companions
+    SELECT TOP 3
+        c.CompanionName,
+        COUNT(ec.CompanionId) AS AppearanceCount
+    FROM tblEpisodeCompanions ec
+    JOIN tblCompanions c ON c.CompanionId = ec.CompanionId
+    GROUP BY c.CompanionName
+    ORDER BY COUNT(ec.CompanionId) DESC;
```

Comment:
if i want to see the episode name along with how many times the companion appeared in all episodes, how can I do that?